plat/mediatek/mt6795/plat_topology.c

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Consider migrating to the new PSCI platform porting interface. See docs/platform-migration-guide.md.